dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -95,7 +95,7 @@ func Export(client *http.Client, conn string, dashboard string, out string) erro
 
 	err = ioutil.WriteFile(out, body, 0666)
 
-	fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out)
+	if ! quiet { fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out) }
```

Comment:
Same here with the newlines.
```
if !quiet {
    fmt.Printf("The dashboard %s was exported under the %s file\n", dashboard, out)
}
```

Reply:
ok, point taken.

Reply:
made changes to indents, removed explicit =='s and removed explict bool